packages/wrangler/src/r2/index.ts

```diff
@@ -51,6 +53,64 @@ export function r2(r2Yargs: CommonYargsArgv) {
 	return r2Yargs
 		.command("object", "Manage R2 objects", (r2ObjectYargs) => {
 			return r2ObjectYargs
+				.command(
+					"filter <bucket> <pattern>",
+					"Filter R2 objects in a bucket",
```

Comment:
How about this:  
```suggestion
					"List objects in an R2 bucket that match a glob pattern",
```

Just because it may be ambiguous what "filtering" means here.